packages/cli/package.json

```diff
@@ -22,22 +22,21 @@
     "@types/prettier": "^2.7.2",
     "@types/recursive-readdir": "^2.2.1",
     "@types/tar-fs": "^2.0.1",
-    "oclif": "3.9.1",
     "tempy": "^3.0.0",
     "type-fest": "^3.6.0",
     "vitest": "^0.33.0"
   },
   "peerDependencies": {
-    "@remix-run/react": "^1.17.1",
+    "@remix-run/react": "^1.19.1",
     "@shopify/hydrogen-react": "^2023.7.0",
     "@shopify/remix-oxygen": "^1.1.1"
   },
   "dependencies": {
     "@ast-grep/napi": "^0.5.3",
     "@graphql-codegen/cli": "3.3.1",
-    "@oclif/core": "2.8.5",
-    "@remix-run/dev": "1.17.1",
-    "@shopify/cli-kit": "3.47.5",
+    "@oclif/core": "2.8.11",
```

Comment:
Because of `generate-manifest.mjs`, can we remove the `@oclif/core` dependency here?